private_facts/src/hello_world/hello_world.py

```diff
@@ -53,19 +40,23 @@ def retrieve_data(self, uri):
 
 tahoe_client = TahoeClient(base_url=BASE_URL)
 
+
 def upload_string(tahoe_client, data):
     """
-    Upload the contents of the test string via the Tahoe client and return its URI.
+    Upload the contents of the test string via tahoe_client and return its URI.
     """
-    uri = tahoe_client.upload_data(data)
-    print(uri)
-    return uri
     
-
+    try:
+        uri = tahoe_client.upload_data(data)
+        print(uri)
```

Comment:
What do you think about something like:
```suggestion
print("Success! Save this capability string to retrieve the content: ", uri)
```